src/browser/deck/actions.js

```diff
@@ -196,6 +196,20 @@ define((require, exports, module) => {
 
   const prepend = (items, item) => insert(items, item, 0);
 
+  const maybeSelectIndex = (items, index) => {
```

Comment:
Just thinking it out loud: It maybe better idea to just update `activate` to:

```js
activate = (items, item=selected(items)) => ....
```

That way `activate` API signature will match `select` API and there will be no need for additional `activateLast` as `goToTab` (btw I'd call it switchToTab`) can be:

```js
const goToTab = index => items => activate(items, items.get(index));
```

Note that if index is out of bound it will return undefined and there for activate selected tab, which is going to be noop.

As of  `alt 9` it will become:

```js
activate(items, items.last());
```